### ISSUE TITLE
Poor compression performance

### ISSUE THREAD

**Issue by umlearn** [mancunianetz/APNGb#5]
Thank you for provide us with this great tool, but at this stage, the compression speed is too slow, and it takes a long time to wait. Running in MacBook Pro 15inch 2017, which got 2.9 GHz Intel Core i7. I don't know why it happened, can you guys fix it?

**Comment by shgodoroja** [comment]
Hi @umlearn 

I'm a bit busy right now, it's hard to tell when this will be under investigation and fixed if so.
But thanks for reporting it.

Stefan.

**Comment by matteoraggi** [comment]
yes it is very super slow also aon a powerfull max xeon octacore 64gb ram and ssd hard disk..
I suggest to add the option to not compress the png too.

**Comment by ValZapod** [comment]
> super slow also aon a powerfu

Yet it is better than what ffmpeg does! So...
